### PR TITLE
Added recipes page

### PR DESCRIPTION
**Pull request by luisdavidgarcia**

### CODE REVIEW

File: react-frontend/src/components/Services/recipe_widget2.js
Comment (Westluu) [inline react-frontend/src/components/Services/recipe_widget2.js:1]: It looks organized and great

**Review by aahmadyar123 — APPROVED** [review]
Good work!

File: react-frontend/src/components/Services/recipe_widget2.js
Comment (Westluu) [reply react-frontend/src/components/Services/recipe_widget2.js:1]: I would also suggest, making it dynamic or mapping it to an array for future use

File: react-frontend/src/pages/recipes.js
Comment (Westluu) [inline react-frontend/src/pages/recipes.js:1]: LGTM

**Review by Westluu — APPROVED** [review]
Good Job! Adding the recipes page, I would just suggest mapping it from an array for future use

**Pull request merged**